lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -91,7 +91,6 @@ def self.keywords
       state :strings do
         rule %r/"/, Str, :pop!
         rule %r/[^"%\n]+/, Str
-        rule %r/(\([a-z0-9_]+\))?[-#0 +]*([0-9]+|[*])?(\.([0-9]+|[*]))?/i, Str
```

Comment:
I just had a complete brain explosion here, didn't I? Not sure why I didn't remove this earlier.

Reply:
eheh, I'm not really sure what it was doing - I looked through it and the previous line definitely catches all this